Packages/com.unity.multiplayer.samples.coop/Utilities/SceneManagement/SceneLoaderWrapper.cs

```diff
@@ -181,7 +180,7 @@ void OnSceneEvent(SceneEvent sceneEvent)
                     if (NetworkManager.IsServer)
                     {
                         // Send client RPC to make sure the client stops the loading screen after the server handles what it needs to after the client finished synchronizing, for example character spawning done server side should still be hidden by loading screen.
-                        StopLoadingScreenClientRpc(new ClientRpcParams { Send = new ClientRpcSendParams { TargetClientIds = new[] { sceneEvent.ClientId } } });
+                        StopLoadingScreenClientRpc(RpcTarget.Group(new[] { sceneEvent.ClientId }, RpcTargetUse.Persistent));
```

Comment:
Do we need to use `RpcTargetUse.Persistent` here, or could we save some GC by using Temp?

According to the documentation:

[Temp](https://docs.unity3d.com/Packages/com.unity.netcode.gameobjects@1.8/api/Unity.Netcode.RpcTargetUse.html#Unity_Netcode_RpcTargetUse_Temp) will return a cached target, which should not be stored as it will be overwritten in future calls to Not() or Group(). Do not call Dispose() on Temp targets.

[Persistent](https://docs.unity3d.com/Packages/com.unity.netcode.gameobjects@1.8/api/Unity.Netcode.RpcTargetUse.html#Unity_Netcode_RpcTargetUse_Persistent) will return a new target, which can be stored, but should not be done frequently because it results in a GC allocation. You must call Dispose() on Persistent targets when you are done with them.

Reply:
@fernando-cortez 
See my comment below about this. Temp would be the better way to go with that use case.